contracts/TypedMemView.sol

```diff
@@ -386,29 +518,57 @@ library TypedMemView {
         }
     }
 
-    /// Parse an unsigned integer from the view at `_index`. Requires that the
-    /// view have >= `_bytes` bytes following that index.
+    /**
+     * @notice          Parse an unsigned integer from the view at `_index`.
+     * @dev             Requires that the view have >= `_bytes` bytes following that index.
+     * @param memView   The view
+     * @param _index    The index
+     * @param _bytes    The bytes
+     * @return          result - The unsigned integer
+     */
     function indexUint(bytes29 memView, uint256 _index, uint8 _bytes) internal pure returns (uint256 result) {
         return uint256(index(memView, _index, _bytes)) >> ((32 - _bytes) * 8);
     }
 
-    /// Parse an unsigned integer from LE bytes.
+    /**
+     * @notice          Parse an unsigned integer from LE bytes.
+     * @param memView   The view
+     * @param _index    The index
+     * @param _bytes    The bytes
+     * @return          result - The unsigned integer
+     */
     function indexLEUint(bytes29 memView, uint256 _index, uint8 _bytes) internal pure returns (uint256 result) {
         return reverseUint256(uint256(index(memView, _index, _bytes)));
     }
 
-    /// Parse a signed integer from the view at `_index`. Requires that the
-    /// view have >= `_bytes` bytes following that index.
+    /**
+     * @notice          Parse a signed integer from the view at `_index`.
+     * @dev             Requires that the view have >= `_bytes` bytes following that index.
+     * @param memView   The view
+     * @param _index    The index
+     * @param _bytes    The bytes
+     * @return          result - The signed integer
+     */
     function indexInt(bytes29 memView, uint256 _index, uint8 _bytes) internal pure returns (int256 result) {
         return int256(index(memView, _index, _bytes)) >> ((32 - _bytes) * 8);
```

Comment:
Seeing error `Explicit type conversion not allowed from "bytes32" to "int256"`, should we be concerned about that?

Reply:
well that's new 👀 

change to this:

`int256(uint256(index(memView, _index, _bytes))) >> ((32 - _bytes) * 8);`

Reply:
actually. better to delete this method entirely. Is it used anywhere?